public/js/cin.js

```diff
@@ -204,7 +209,7 @@ const ScheduleArea = () => {
                         ) != -1
                     ) {
                         if (new Date(event.end_time).getTime() < Date.now()) {
-                            var youtube = event.interview.youtube_id;
+                            let youtube = event.interview.youtube_id;
```

Comment:
As discussed on Slack, CIN-API might sometimes give empty strings

```suggestion
                            let youtube = event.interview.youtube_id;
                            if (typeof youtube === "string" && youtube.length === 0 {
                                youtube = null;
                            }
```